server/json/user_json.jsx

```diff
@@ -18,6 +18,7 @@ export default function useUserJson(app) {
 
         if(chainAccount) {
             user = chainAccount;
+            user.json_metadata = user.json_metadata ? JSON.parse(user.json_metadata) : "";
```

Comment:
@sneak This conditional will actually catch for users with/without JSON metadata.  If the user has no JSON it won't parse/empty string.  In the case of malformed data it would seem the backend should be validating on the save. But I added the try/catch.